tests/test_csv.py

```diff
@@ -68,14 +68,14 @@ def test_custom_sql_csv(app_client):
         "/fixtures.csv?sql=select+content+from+simple_primary_key+limit+2"
     )
     assert response.status == 200
-    assert "text/plain; charset=utf-8" == response.headers["Content-Type"]
+    assert "text/plain; charset=utf-8" == response.headers["content-type"]
```

Comment:
I'm not entirely happy with these changes regarding header case-insensitivity. It would be good to have a header abstraction which dealt with case-insensitivity in a nicer way.

Reply:
Filed as a separate issue: #522